enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -123,6 +123,10 @@ func (i *InvocationStatService) GetTrend(ctx context.Context, req *inpb.GetTrend
 		q.AddWhereClause("branch_name = ?", branchName)
 	}
 
+	if command := req.GetQuery().GetCommand(); command != "" {
```

Comment:
"command" was not added as a column in clickhouse yet. 
https://cs.github.com/buildbuddy-io/buildbuddy/blob/6e1e97fcf1d1b435672ad43e4616ea63ff6fae29/server/util/clickhouse/clickhouse.go#L50

This means when we switch on clickhouse the query will be broken because the column doesn't exist. maybe only add the where clause when  i.isOLAPDBEnabled() is false.

Reply:
Added the column to clickhouse - definitely want this to work in either case.

Would there be some sort of backfill that I need to run? Or have we not done that in production yet?